frontend/src/components/GifWidget/GifWidget.styled.ts

```diff
@@ -0,0 +1,89 @@
+import styled from "styled-components";
+
+const GifWidgetStyled = styled.div`
+    position: absolute;
+    right: 3rem;
+    bottom: 7rem;
+
+    width: 30rem;
+    height: 35rem;
+    background-color: #000000bb;
+
+    box-shadow: 5px 10px 25px #000000bb;
+    border-radius: 4px;
+    overflow-y: auto;
+
+    &::-webkit-scrollbar {
+        width: 5px;
+    }
+
+    &::-webkit-scrollbar-thumb {
+        border-radius: 3px;
+        background: ${({ theme }) => theme.primary};
+    }
+`;
+
+const GifWidgetContent = styled.div``;
```

Comment:
Empty

Reply:
Ye, remove it, if have no style, use "div" over it